### ISSUE TITLE
Progress of HTTP-Update

### ISSUE THREAD

**Issue by Disane87** [esp8266/Arduino#4183]
Referecing the example for the HTTP-Update: https://github.com/esp8266/Arduino/libraries/ESP8266httpUpdate/examples/httpUpdate/httpUpdate.ino

Is there any possibility to get the current status of the download and flash progress?
I want to show a progressbar with the current percentage. 

Any chance to achive this?

**Comment by CarlosJimenez** [comment]
I tried this (present in the latest version of Update.cpp) and it works for me!!:

```cpp
void progressFunc(unsigned int progress,unsigned int total) {
  Serial.printf("Progress: %u of %u\r", progress, total);
};
Update.onProgress(progressFunc);
```

**Comment by earlephilhower** [comment]
Closing as fixed via #6796
